devsupport/.gitignore

```diff
@@ -0,0 +1,3 @@
+db/
```

Comment:
Please remove these `devsupport` files.

Reply:
See https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b for how to ignore these files for you locally.